docs/source/notebooks/SMC2_gaussians.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>I still don't understand why we have to do that, but I think it's because I don't know the litterature on SMC well enough yet, so I trust you!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3981/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
You can use a potential with any sampler, this is not something related to SMC. By defining a potential you are sampling exactly from the intended "distribution", if you use a likelihood this only happens in the limit of infinite data.

Reply:
<blockquote>By defining a potential you are sampling exactly from the intended "distribution", if you use a likelihood this only happens in the limit of infinite data.</blockquote><p>Oooh ok, I didn't know that! Thanks for your patience, I get it know 😅</p>